.gitmodules

```diff
@@ -20,4 +20,4 @@
 [submodule "windows_docker_resources/ros2-cookbooks"]
 	path = windows_docker_resources/ros2-cookbooks
 	url = git@github.com:ros-infrastructure/ros2-cookbooks
-	branch = latest
+	branch = clalancette/add-humble
```

Comment:
> I guess this will have to be updated again before merging, right?

Yes, exactly.  Given that I have two approvals here, I'm going to go ahead and merge this (without this bit), then deploy.